package.json

```
@@ -6,10 +6,8 @@
     "build": "run-s build:*",
     "build:remix": "remix build",
     "build:server": "esbuild --platform=node --format=cjs ./server.ts --outdir=build --bundle",
-    "dev": "run-p dev:*",
-    "dev:build": "cross-env NODE_ENV=development npm run build:server -- --watch",
-    "dev:remix": "cross-env NODE_ENV=development remix watch",
-    "dev:server": "cross-env NODE_ENV=development node --inspect --require ./node_modules/dotenv/config --require ./mocks ./build/server.js",
+    "dev": "remix dev -c \"npm run dev:serve\"",
+    "dev:serve": "binode --require ./mocks -- @remix-run/serve:remix-serve ./build/server.js",
```

This stack directly runs the server with `node` without needing `remix-serve`:

```suggestion
    "dev:serve": "node --require ./mocks ./build/server.js",
```